package/MDAnalysis/analysis/pca.py

```diff
@@ -414,6 +414,42 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         return dot
 
+    def project_single_frame(self, components=None):
+        """Runs an inverse-PCA and generates
+           a trajectory that is projected on selected PCs
+
+        Run PCA class before using this function.
+        To project the trajectory onto the first principal component,
+            pca = PCA(universe, select='backbone').run()
+            func = pca.project_single_frame(components=0)
+
+        To apply the projection on-the-fly, may add as a transformation
+            u.trajectory.add_transformations(func)
```

Comment:
The synopsis should describe in words what the method does.

Then add an 
```resT
Example
---------
```
section after the *Returns* section.

See the general structure of docs https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard